### ISSUE TITLE
Error pop up

### ISSUE THREAD

**Issue by aloisdg** [giacmir/Inkscape-JPEG-export-extension#4]
Hello,

I was trying to use your extension on Windows 8.1. When something wrong happened :

![error inkscape](https://cloud.githubusercontent.com/assets/3449303/8180891/d1eef8fc-1421-11e5-9884-29ec75341236.PNG)

Full copy of the error:

```
Traceback (most recent call last):
  File "jpegexport.py", line 175, in <module>
    _main()
  File "jpegexport.py", line 171, in _main
    e.affect()
  File "C:\Program Files\Inkscape\share\extensions\inkex.py", line 268, in affect
    self.effect()
  File "jpegexport.py", line 56, in effect
    coords=self.processSelected()
  File "jpegexport.py", line 75, in processSelected
    toty=inkex.unittouu(root.attrib['height'])
AttributeError: 'module' object has no attribute 'unittouu'
```

What can I do?

**Comment by giacmir** [comment]
Thanks for you report

It seems that `unittouu` is not part of the Inkex module anymore. I will take a look on this as soon as possible.

**Comment by su-v** [comment]
@giacmir - unittouu is still part of inkex.py in Inkscape 0.91, but it was moved to the inkex.Effect class:
http://wiki.inkscape.org/wiki/index.php/Release_notes/0.91#Units:_Breaking_change

**Comment by aloisdg** [comment]
@giacmir You are welcome. :)

**Comment by giacmir** [comment]
Just pushed a new version with fix for this issue; I tested with Inkscape 0.91 r13725. Please test it and let me know if it works.

**Comment by giacmir** [comment]
No direct feedback but 61e4497b75d53a62887f9265902e5bfa9aeb4a16 should have fixed. Please reopen if needed.
